internal/starlark/starlark.go

```diff
@@ -0,0 +1,150 @@
+package starlark
+
+import (
+	"fmt"
+	"github.com/starlight-go/starlight/convert"
+	"go.starlark.net/starlark"
+	"math"
+	"net/url"
+	"regexp"
+)
+
+type StarlarkFunctionHandle struct {
+	Code   string
+	State  starlark.StringDict
+	API    map[string]interface{}
```

Comment:
I think commonly this would be called "env" or something, but i wonder if we should instead just make this a separate starlark package "the aft starlark stdlib" basically and import that